Automattic/pocket-casts-android
# [Give Rating] Adding whats new for Give Rating

## Description
- This PR adds the give rating for whats new with animation. 
- The initial state should show no stars and then play the animation.
- See: p1722454541296259/1722365320.671129-slack-C077XU4GF9D

Fixes #2543 


## Testing Instructions
1. Make give rating feature flag is enabled
2. Apply this [enable_whats_new.patch](https://github.com/user-attachments/files/16470652/enable_whats_new.patch)
3. Run the App
4. Check the whats new modal for give rating


## Screenshots or Screencast 
![screen-20240805-150228-ezgif com-crop](https://github.com/user-attachments/assets/2bafb707-18b9-4b2e-87ce-520b7ef6017e)

https://github.com/user-attachments/assets/bb5d2caa-067d-46bd-9da1-ee994f85dc87


## Checklist
 
#### I have tested any UI changes...
